libraries/lwIP_enc28j60/src/utility/enc28j60.h

```diff
@@ -41,6 +41,35 @@
 
 #include <SPI.h>
 
+// PHLCON: PHY module LED control register
+//
+//      R R R R    L  L  L  L    L  L  L  L             R
+//      E E E E    A  A  A  A    B  B  B  B    L  L  S  E
+//      S S S S    C  C  C  C    C  C  C  C    F  F  T  S
+//      E E E E    F  F  F  F    F  F  F  F    R  R  R  E
+//      R R R R    G  G  G  G    G  G  G  G    Q  Q  C  R
+//      V V V V    3  2  1  0    3  2  1  0    1  0  H  V
```

Comment:
Can this table receive more comments on what it is for ?

Reply:
This is LED control register that described in the ENC28J60 document (page 11) and used with LEDsConfig function

usage:
`LEDsConfig(LED_LINK_STATUS, LED_TRANS_RECV);                     // LEDA, LEDB`
`LEDsConfig(LED_BLINK_SLOW, LED_NO_CHANGE);                     // LEDA, LEDB`

http://ww1.microchip.com/downloads/en/devicedoc/39662a.pdf

Reply:
Do you plan to add this informations as source-code comments in your proposal ?